spec/methods/auth/auth.ts

```diff
@@ -339,5 +342,200 @@ export namespace ConfirmPassReset {
    */
   export type Result = MessageResult;
 
+  export type Response = MessageResponse;
+}
+
+//////////////////////////////////
+// Configure MFA Types
+//////////////////////////////////
+
+/**
+ * Response from a call to begin
+ * App MFA setup.  It contains a secret code
+ * to enter into the MFA App.
+ */
+export type SecretCodeResult = {
+  secretCode: string
+};
+
+/**
+ * Decoded API response from a call to begin
+ * App MFA setup.  It contains a secret code
+ * to enter into the MFA App.
+ */
+export type SecretCodeResponse = ApiResponse<SecretCodeResult>;
+
+/**
+ * Sets user MFA preferences
+ */
+export namespace SetMfaPreference {
+
+  export const HTTP:HttpMethods.POST = 'POST';
+  export const Path = `${authBasePath}/${ResourcePaths.configureMfa}`;
+
+  export interface Args {
+    username: string,
```

Comment:
We can drop this value, actually, as it's an authenticated API and we get their `email` for free.